components/x-gift-article/src/Form.jsx

```diff
@@ -4,17 +4,18 @@ import RadioButtonsSection from './RadioButtonsSection';
 import UrlSection from './UrlSection';
 import MobileShareButtons from './MobileShareButtons';
 import CopyConfirmation from './CopyConfirmation';
-import styles from './GiftArticle.css';
+import styles from './GiftArticle.scss';
 
 const formClassNames = [
-	'o-forms',
-	styles.form
+	styles["share-form"]
```

Comment:
Stylistic point feel free to ignore, but now this is just one class I'd just update line 15 below to be `className={styles['share-form']}` rather than having an intermediary variable.

Reply:
Ah yes - I had done that elsewhere but missed this